dlt/extract/incremental/typing.py

```diff
@@ -1,8 +1,9 @@
-from typing import TypedDict, Optional, Any, List, TypeVar, Callable, Sequence
+from typing import TypedDict, Optional, Any, List, Literal, TypeVar, Callable, Sequence
 
 
 TCursorValue = TypeVar("TCursorValue", bound=Any)
 LastValueFunc = Callable[[Sequence[TCursorValue]], Any]
+OnCursorValueNone = Literal["raise", "include"]
```

Comment:
why no "exclude"? this OFC can be done with add_map.

Reply:
Yes, I didn't want to duplicate the functionality of add_filter(). But, since we already have one parameter here I could add that too without much trouble.